R/mail.R

```diff
@@ -0,0 +1,54 @@
+# Kaiaulu - https://github.com/sailuh/kaiaulu
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+
+
+
+#' Creates fake mbox data
+#'
```

Comment:
There is no description body on the function. Please add a bit more here what this function is for! See the git fake data as example. Unless I forgot to point that out there too :^)